CMakeLists.txt

```diff
@@ -112,29 +112,47 @@ include_directories(${PROJECT_BINARY_DIR})
 
 # static library
 
+option(BUILD_SHARED "Build shared library." ON)
 option(BUILD_STATIC "Build static library." OFF)
 
+if (NOT BUILD_STATIC AND NOT BUILD_SHARED)
+    message(FATAL_ERROR "BUILD_SHAREDand BUILD_STATIC cannot be both disabled")
```

Comment:
There a typo, a space character missing between `BUILD_SHARED` and `and`

Reply:
This is fixed.